canonical/canonical-design
# WD-16708 Contact us and Error templates

## Done
- Added templates for contact-us page and error pages
- Since the contact page is static we have added a dedicated thank-you page.
- Made footer sticky 
- Added Djlint

## QA

- Try filling up contact-us form and see if information is being recorded and the form is being submitted properly.
- Try accessing pages that don't exist or create errors that would spin up required templates to see if they work properly.

## Fixes

Jira Issue - [WD-16708](https://warthogs.atlassian.net/browse/WD-16708)
